frontend/src/lib/stores/childrenStore.svelte.ts

```diff
@@ -0,0 +1,47 @@
+import { type ChildPublic, getChild } from "$lib/client";
+
+function createCurrentChild() {
+	let currentChild: number | null = $state(null);
+	let childdata: ChildPublic | null = $state(null);
+	return {
+		get id() {
+			return currentChild;
+		},
+		get name() {
+			return childdata?.name;
+		},
+		get month() {
+			return childdata?.birth_month;
+		},
+		get year() {
+			return childdata?.birth_year;
+		},
```

Comment:
is there a benefit to adding these getters vs just providing access to the ChildPublic object?

Reply:
I personally find it more intuitive to be able to write child.name, child.month etc instead of child.data.month, and b/c I don´t see a reason why the data model should change it doesn´t seem to be a maintenance dept.